File: Python Word Examples/ParagraphsApiDemo/getParagraphChildObjects.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ParagraphsApi(configuration)

file_name = "Template.docx"
node_path = "Section/0/Body/0"
index = 5
result = api.get_paragraph_child_objects(file_name, node_path, index, folder="input")

File: PHP Word Examples/ParagraphsApiDemo/getParagraphChildObjects.php
<?php

use \Spire\Cloud\Word\Sdk\Client\Api\ParagraphsApi;
use \Spire\Cloud\Word\Sdk\Client\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new ParagraphsApi($configuration);
$fileName = "sample1.docx";
$node_path = 'Section/0/Body/0';
$index = 0;
$folder = "input";
$storage = null;
$password = null;
$result = $apiInstance->getParagraphChildObjects($fileName, $node_path, $index, $folder, $storage, $password);
print_r($result);